scripts/core/exp-bindless-images.yml

```diff
@@ -285,6 +291,42 @@ name: $x_exp_interop_semaphore_desc_t
 base: $x_base_desc_t
 members: []
 --- #--------------------------------------------------------------------------
+type: struct
+desc: "Provides information required for copying to/from non-standard images"
+name: $x_exp_image_copy_subresource_t
+base: $x_base_desc_t
+members:
+    - type: uint32_t
+      name: mipLevel
+      desc: "[in] the mipmap level to copy"
+    - type: uint32_t
+      name: baseArrayLayer
+      desc: "[in] the starting image layer to copy"
+    - type: uint32_t
+      name: layerCount
+      desc: "[in] number of layers to copy"
+--- #--------------------------------------------------------------------------
```

Comment:
Looking at this again, it seems like this isn't used anywhere. Can we remove this from the PR?

Reply:
removed.

Reply:
Isaac is currently unavailable, but we've confirmed with the rest of the team that the current patch addresses this comment.